scripts/versions.mjs

```diff
@@ -1,10 +1,15 @@
 import { exec } from 'node:child_process';
+import { dirname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
 import { promisify } from 'node:util';
 
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const rootDir = join(__dirname, '..');
 const execAsync = promisify(exec);
 
 const fileToESMWrapperCall = (path, version) =>
-	execAsync(`npx gen-esm-wrapper ./${path}/${version}.js ./${path}/${version}.mjs`);
+	execAsync(`npx gen-esm-wrapper ${rootDir}${path}/${version}.js ${rootDir}${path}/${version}.mjs`);
```

Comment:
You could use `join` for building OS-normalized paths, and you should wrap them between quotes, or else this script will fail when the project is inside a folder with spaces.